### PR TITLE
DSDEEPB-1929: use dropdown for root entity type

### PR DESCRIPTION
**Pull request by davidangb**
Manually tested:
* editing a config that pre-existed in your workspace
* importing a method from agora
* importing a config from agora

(note that in the last use case, you do not have the option of changing the root entity type; I tested this to ensure there was no regression)

At the code level, I don't like having both style/create-select and style/create-identity-select; these could be refactored back into a single function, but doing so opens a large-ish risk of regression.

### CODE REVIEW

File: src/cljs/org/broadinstitute/firecloud_ui/common.cljs [hunk 1]
```diff
@@ -131,3 +131,6 @@
   (when-not (clojure.string/blank? segment)
     (let [[ns n] (clojure.string/split segment #":")]
       {:namespace ns :name n})))
+
+(def root-entity-types
+  '("participant","sample","pair","participant_set","sample_set","pair_set"))
```
Comment (MattPutnam) [inline src/cljs/org/broadinstitute/firecloud_ui/common.cljs]: Can this just be a vector (not a list)?  Also, drop the commas.

File: src/cljs/org/broadinstitute/firecloud_ui/page/method_config_importer.cljs [hunk 2]
```diff
@@ -51,10 +51,15 @@
...
          [:div {:style {:float "left" :marginRight "0.5em"}}
-          (style/create-form-label (:label field))
-          [input/TextField {:defaultValue (entity (:key field))
-                            :ref (:key field) :placeholder "Required"
-                            :predicates [(input/nonempty "Fields")]}]])
+            (style/create-form-label (:label field))
+              (cond (= (:type field) "identity-select")
+                (style/create-identity-select {:ref (:key field)
+                  :value (entity (:key field))}
+                  root-entity-types)
```
Comment (MattPutnam) [inline src/cljs/org/broadinstitute/firecloud_ui/page/method_config_importer.cljs]: Should this be `(:options field)`?

**Comment by MattPutnam** [comment]
Check indentation throughout.  Otherwise, just two little things.

File: src/cljs/org/broadinstitute/firecloud_ui/page/method_config_importer.cljs (hunk 2 above)
Comment (davidangb) [inline src/cljs/org/broadinstitute/firecloud_ui/page/method_config_importer.cljs]: yes. yes it should.

**Comment by davidangb** [comment]
back to you, Matt, and thanks!

**Comment by MattPutnam** [comment]
:+1:

**Pull request merged**